templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <button type="submit" class="btn btn-danger">Yes</button>
+    <a href="{% url "taxi:driver-detail" pk=driver.pk %}" class="btn btn-outline-danger">No</a>
+  </form>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on